src/AccessibilityInsights.SharedUx/FileIssue/IssueReporter.cs

```diff
@@ -80,7 +80,8 @@ public static IIssueResult FileIssueAsync(IssueInformation issueInformation)
                 // It does seem like we currently block the main thread when we show the win form for azure devops
                 // so keeping it as is till we have a discussion. Check for blocking behavior at that link.
                 // https://github.com/Microsoft/accessibility-insights-windows/blob/main/src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml.cs#L858
-                IIssueResult result = IssueReporting.FileIssueAsync(issueInformation).Result;
+                IIssueResultWithPostAction result = IssueReporting.FileIssueAsync(issueInformation).Result;
+                result.PostAction?.Invoke();
                 IssueReporterManager.GetInstance().UpdateIssueReporterSettings(IssueReporting);
```

Comment:
@DaveTryon 
I think we can still update the settings if there's a PostAction. But if we felt otherwise, we could have the action return a boolean that indicates whether to persist settings, or the extensions can manage this themselves (they can return the same or null settings to no-op the update). My feeling is that we still want to hold on to their server & team info, even if the attachment failed (could be as simple as a mangled GUID in the preview).